astropy/io/fits/__init__.py

```diff
@@ -84,20 +84,17 @@ class Conf(_config.ConfigNamespace):
 from .header import Header
 from .verify import VerifyError
 
-__all__ = (
-    ["Conf", "conf"]
-    + card.__all__
-    + column.__all__
-    + convenience.__all__
-    + hdu.__all__
-    + [
-        "FITS_record",
-        "FITS_rec",
-        "GroupData",
-        "open",
-        "Section",
-        "Header",
-        "VerifyError",
-        "conf",
-    ]
-)
+__all__ = ["Conf", "conf"]
+__all__ += card.__all__
+__all__ += column.__all__
+__all__ += convenience.__all__
+__all__ += hdu.__all__
+__all__ += [
+    "FITS_record",
+    "FITS_rec",
+    "GroupData",
+    "open",
+    "Section",
+    "Header",
+    "VerifyError",
+]
```

Comment:
This could be accomplished in the same way I showed above

Reply:
Edit: this current change is not what RUF005 suggests, but what ``python.typing`` suggests to enable static analysis of ``__all__``.